static/app/views/settings/organizationIntegrations/docIntegrationDetailedView.tsx

```diff
@@ -1,88 +1,67 @@
 import styled from '@emotion/styled';
 
 import DocIntegrationAvatar from 'sentry/components/avatar/docIntegrationAvatar';
 import {Button} from 'sentry/components/button';
-import type DeprecatedAsyncComponent from 'sentry/components/deprecatedAsyncComponent';
 import ExternalLink from 'sentry/components/links/externalLink';
+import LoadingError from 'sentry/components/loadingError';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
 import {IconOpen} from 'sentry/icons';
 import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 import type {DocIntegration} from 'sentry/types/integrations';
-import withOrganization from 'sentry/utils/withOrganization';
-
-import AbstractIntegrationDetailedView from './abstractIntegrationDetailedView';
-
-type Tab = AbstractIntegrationDetailedView['state']['tab'];
-
-type State = {
-  doc: DocIntegration;
-};
-
-class DocIntegrationDetailedView extends AbstractIntegrationDetailedView<
-  AbstractIntegrationDetailedView['props'],
-  State & AbstractIntegrationDetailedView['state']
-> {
-  tabs: Tab[] = ['overview'];
-
-  getEndpoints(): ReturnType<DeprecatedAsyncComponent['getEndpoints']> {
-    const {
-      params: {integrationSlug},
-    } = this.props;
-    return [['doc', `/doc-integrations/${integrationSlug}/`]];
-  }
-
-  get integrationType() {
-    return 'document' as const;
-  }
-
-  get integration(): DocIntegration {
-    return this.state.doc;
-  }
-
-  get description() {
-    return this.integration.description;
-  }
-
-  get author() {
-    return this.integration.author;
-  }
-
-  get resourceLinks() {
-    return this.integration.resources ?? [];
-  }
-
-  get installationStatus() {
-    return null;
-  }
-
-  get integrationName() {
-    return this.integration.name;
-  }
-
-  get featureData() {
-    return this.integration.features ?? [];
-  }
-
-  get requiresAccess() {
-    return false;
-  }
-
-  componentDidMount() {
-    super.componentDidMount();
-    this.trackIntegrationAnalytics('integrations.integration_viewed', {
+import {trackIntegrationAnalytics} from 'sentry/utils/integrationUtil';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import useOrganization from 'sentry/utils/useOrganization';
+import {useParams} from 'sentry/utils/useParams';
+import type {Tab} from 'sentry/views/settings/organizationIntegrations/abstractIntegrationDetailedView';
+import IntegrationLayout from 'sentry/views/settings/organizationIntegrations/detailedView/integrationLayout';
+
+export default function DocIntegrationDetailsView() {
+  const tabs: Tab[] = ['overview'];
+  const organization = useOrganization();
+  const {integrationSlug} = useParams<{integrationSlug: string}>();
+
+  const {data: doc, isPending} = useApiQuery<DocIntegration>(
+    [`/doc-integrations/${integrationSlug}/`],
+    {staleTime: Infinity, retry: false}
+  );
+
+  const integrationType = 'document';
+  const description = useMemo(() => doc?.description ?? '', [doc]);
```

Comment:
Are these useMemos necessary? The ones that return strings shouldn't have any benefits, and the ones that return objects would only really benefit if something downstream was memoized or depended on it for a useeffect